modules/search/src/main/java/com/sygic/maps/module/search/callback/SearchResultCallback.kt

```diff
@@ -39,4 +41,9 @@ interface SearchResultCallback {
      *
      */
     fun onSearchResult(searchResultList: List<SearchResult>)
+}
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+interface SearchResultCallbackWrapper {
```

Comment:
viem ze ma hnevalo ked som to robil, ze to nemohlo byt nepublic. ale ty toto mas v 1 module pozeram, mohlo by to byt aj internal, nie?

Reply:
pravdu máš, upravím